docs/debugging.md

```diff
@@ -14,97 +14,46 @@ React Native provides an in-app developer menu which offers several debugging op
-:::
-
-:::tip
+- Select "Open DevTools" in the Dev Menu.
+- Press <kbd>j</kbd> from the CLI (`npx react-native start`).
 
-#### Alternative debugging tools
+On first launch, DevTools will open to a welcome panel, along with an open console drawer where you can view logs and interact with the JavaScript runtime. From the top of the window, you can navigate to other panels, including the integrated React DevTools Components Inspector and Profiler.
 
-As React Native transitions away from Flipper, we recommend other existing methods, including first party IDEs, to inspect your application's native code and behaviour.
+![React Native DevTools opened to the "Welcome" pane](/docs/assets/debugging-rndt-welcome.jpg)
 
-- [Debugging Native Code](./debugging-native-code)
-- <a href="https://shift.infinite.red/why-you-dont-need-flipper-in-your-react-native-app-and-how-to-get-by-without-it-3af461955109" target="_blank">Why you don’t need Flipper in your React Native app … and how to get by without it ↗</a>
-
-:::
+React Native DevTools is powered by a dedicated debugging architecture built into React Native and uses a customized build of the [Chrome DevTools](https://developer.chrome.com/docs/devtools) frontend. This enables us to offer familiar, browser-aligned debugging features that are deeply integrated and built for end-to-end reliability.
 
-</TabItem>
-<TabItem value="new-debugger">
+Learn more in our [React Native DevTools guide](./react-native-devtools).
 
 :::note
-**This is an experimental feature** and several features may not work reliably today. When this feature does launch in future, we intend for it to work more completely than the current debugging methods.
+React Native DevTools is only available with the Hermes engine, and requires either Google Chrome or Microsoft Edge installed.
 :::
 
-The React Native team is working on a new JavaScript debugger experience, intended to replace Flipper, with a preview available as of React Native 0.73.
-
-The new debugger can be enabled via React Native CLI. This will also enable <kbd>j</kbd> to debug.
-
-```sh
-npx react-native start --experimental-debugger
-```
-
-When selecting **"Open Debugger"** in the Dev Menu, this will launch the new debugger using Google Chrome or Microsoft Edge.
-
-![The new debugger frontend opened to the "Welcome" pane](/docs/assets/debugging-debugger-welcome.jpg)
-
-</TabItem>
-</Tabs>
-
-## React DevTools
+:::info
 
-You can use React DevTools to inspect the React element tree, props, and state.
+#### Flipper and alternative debugging tools
 
-```sh
-npx react-devtools
-```
+React Native DevTools replaces the previous previous Flipper, Experimental Debugger, and Hermes debugger (Chrome) entry points. If you are on an older version of React Native, please go to the docs [for your version](/versions).
```

Comment:
```suggestion
React Native DevTools replaces Flipper, the Experimental Debugger, and the Hermes debugger (Chrome). If you are on an older version of React Native, please go to the docs [for your version](/versions).
```

Reply:
[Lower conviction comment!] Part of this phrasing is to not overly identify an "Experimental Debugger" or "Hermes debugger" as such tangible things. Instead, I'm aiming to highlight that these were entry points to the same thing — the previous Hermes debugger backend.

If we can, specific wording here maintains accuracy, while still accessible to developer audience.

Reply:
What if we say that explicitly then:

> React Native DevTools replaces the entry points for Flipper, the Experimental Debugger, and the Hermes Debugger (Chrome).

WDYT?

Reply:
How about "[...] frontends."?

(And just noticed the double "previous" 😬)